ios/AppDelegate.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  AppDelegate.swift
+//  NewExpensify
+//
+//  Created by Marcin Warchoł on 08/04/2025.
+//
+
+import UIKit
+import React
+import React_RCTAppDelegate
+import ReactAppDependencyProvider
+import ExpoModulesCore
+import Firebase
+
+ 
+@main
+class AppDelegate: ExpoAppDelegate, UNUserNotificationCenterDelegate {
```

Comment:
Why do we extend other class now? Is `ExpoAppDelegate` the swift counterpart of `EXAppDelegateWrapper`?